platform-tests/src/test/java/org/junit/platform/engine/support/descriptor/FilePositionTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.platform.engine.support.descriptor;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * @since 1.0
+ */
+class FilePositionTests{
+
+	
+	void equalsTest() {
+		//test for a return of true when the equals method is passed itself
+		FilePosition file = new FilePosition(0, 0);
+		assertEquals(file, file);
+	}
+
+	
+	void equalsNullTest() {
+		//test for a return of true when the equals method is passed null
+		FilePosition file = new FilePosition(0, 0);
+		assertNotNull(file);
```

Comment:
This assertion is meaningless. If the previous line succeeds (i.e., does not throw an exception), then the `file` reference can never be `null`.